salt/renderers/pass.py

```diff
@@ -45,6 +45,30 @@
 
         pass:
           pkg.installed
+
+Configuration options
```

Comment:
Can you clarify for the user which config file these config options need to be set.

Reply:
I've changed `Configuration options` to `Salt master configuration options`